src/bump/performBump.ts

```diff
@@ -44,7 +57,7 @@ export async function performBump(bumpInfo: BumpInfo, options: BeachballOptions)
 
   await callHook('prebump', bumpInfo, options);
 
-  writePackageJson(modifiedPackages, packageInfos);
+  writePackageJson(modifiedPackages, packageInfos, options.path);
```

Comment:
I would much prefer to have this be a separated function so we can dedicate "writePackageJson" to just writing the package.json. And then we can have a post-bump lock file update kind of function.